### DISCUSSION TITLE
Capture 3 levels of hazard severity as high/medium/low and capture probabilities of asset damage (H/M/L). Enable values for the three levels to be assigned by the user #51

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#132] [General]
Need to discuss this further so we have more direction on how we will implement

Part of Asset Inventory: Capturing the perils to which each asset is exposed

**Comment by luke-feeney** [comment]
Schema extension - TerminusDB @KittyJose @GavinMendelGleason
